### PR TITLE
Placeholder Dependent Status Check added.

### PR DESCRIPTION
**Pull request by CanadianPhilipBarkow**
### Description

This is placeholder based on the existing status check.

### Related Azure Boards Work Items

[AB#3366](https://dev.azure.com/DTS-STN/Canada%20Dental%20Care%20Plan/_workitems/edit/3366)

### Screenshots (if applicable)

Screenshot from 2024-05-10 10-56-58

Screenshot from 2024-05-10 10-56-19

### Checklist

- [X] I have tested the changes locally
- [X] I have added/updated tests that prove my fix is effective or that my feature works
- [X] I have checked that my code follows the project's coding style by running `npm run format:check`
- [X] I have checked that my code contains no linting errors by running `npm run lint`
- [X] I have checked that my code contains no type errors by running `npm run typecheck`
- [X] I have checked that all unit tests pass by running `npm run test:unit -- run`
- [X] I have checked that all e2e tests pass by running `npm run test:e2e`

### Test Instructions

1. Include `dependent-status-checker` in the list of `ENABLED_FEATURES` in the .env file
2. Open the protected home page
3. Confirm the presence of the Dependent Status Checker card
4. Click the Dependent Status Checker card
5. Confirm that the Dependent Status Checker placeholder is displayed

### CODE REVIEW

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]
Rename `check` to `checker` everywhere. Don't forget the filenames.

**Comment by CanadianPhilipBarkow** [comment]
> Rename `check` to `checker` everywhere. Don't forget the filenames.

Sure. Implemented in commit 360f759.

File: frontend/app/routes/$lang+/_protected+/dependent-status-checker+/index.tsx [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+export async function loader({ context: { session }, params, request }: LoaderFunctionArgs) {
+  featureEnabled('dependent-status-checker');
+
+  const raoidcService = await getRaoidcService();
+  await raoidcService.handleSessionValidation(request, session);
+  const instrumentationService = getInstrumentationService();
+
+  const idToken: IdToken = session.get('idToken');
+  getAuditService().audit('page-view.status-check', { userId: idToken.sub });
```
Comment (sebastien-comeau) [inline frontend/app/routes/$lang+/_protected+/dependent-status-checker+/index.tsx:39]:
```suggestion
  getAuditService().audit('page-view.dependent-status-checker', { userId: idToken.sub });
```

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]

**Review by sebastien-comeau — DISMISSED** [review]

File: frontend/app/routes/$lang+/_protected+/dependent-status-checker+/index.tsx (hunk 1 above)
Comment (CanadianPhilipBarkow) [reply frontend/app/routes/$lang+/_protected+/dependent-status-checker+/index.tsx]: Thanks for catching that. Implemented in commit ade45087bcec0267b9518c0e361aae2000367100.

Comment (sebastien-comeau) [reply frontend/app/routes/$lang+/_protected+/dependent-status-checker+/index.tsx]: `dependent-status-checker` and not `dependent-status-check`

Comment (CanadianPhilipBarkow) [reply frontend/app/routes/$lang+/_protected+/dependent-status-checker+/index.tsx]: Fixed in commit 48b8a0c4a1b5742aa8d72ae86f64379a7096e38a

**Review by sebastien-comeau — APPROVED** [review]

**Pull request merged**